vsts/test-release-nuget.yaml

```diff
@@ -89,6 +86,7 @@ jobs:
           LA_AAD_APP_ID: $(LA-AAD-APP-ID)
           LA_AAD_APP_CERT_BASE64: $(LA-AAD-APP-CERT-BASE64)
           LA_WORKSPACE_ID: $(LA-WORKSPACE-ID)
+          E2E_IKEY: $(E2E-IKEY)
```

Comment:
What is this used for? I don't see the e2e tests looking for this value

Reply:
This is being used to hook up our tests to application insights: https://github.com/Azure/azure-iot-sdk-csharp/blob/master/e2e/test/Helpers/logging/TestLogger.cs#L40